config/manifests/xdescriptors_patch.yaml

```diff
@@ -1,6 +1,7 @@
-- op: replace
-  path: /spec/customresourcedefinitions/owned/2/specDescriptors/3/path
-  value: flightRecorder.name
 - op: add
-  path: /spec/customresourcedefinitions/owned/2/specDescriptors/3/x-descriptors
-  value: [ "urn:alm:descriptor:io.kubernetes:operator.cryostat.io:v1beta1:FlightRecorder" ]
+  path: /spec/customresourcedefinitions/owned/2/specDescriptors/3
+  value:
+    description: Select the FlightRecorder with the name of the target Pod for this Recording.
+    displayName: Name
+    path: flightRecorder.name
+    x-descriptors: [ "urn:alm:descriptor:io.kubernetes:operator.cryostat.io:v1beta1:FlightRecorder" ]
```

Comment:
Could you re-save this file with the newline at the end of it? Some editors do not do this by default. If using VSCode, you can enable this behaviour by searching settings for 'Insert Final Newline', and checking it.

Reply:
yes I was looking to how to do this! Thanks a lot! Just configured and added newline!